requirements.yml

```diff
@@ -6,8 +6,8 @@ dependencies:
   - conda-forge::matplotlib
   - conda-forge::descartes
   - conda-forge::pandas
-  - conda-forge::numpy<1.24
+  - conda-forge::numpy<=1.24
   - conda-forge::scipy
   - conda-forge::scikit-learn
-  - conda-forge::geokit>=1.4.0
-  - conda-forge::gdal=3.4.*
+  - conda-forge::gdal>2.4,<=2.4.4
+  - conda-forge::geokit=1.3.0
```

Comment:
Is it necessary to fix geokit 1.3.0 exactly or could this be changed to >=1.3.0? @tpelser

Reply:
yes this is necessary due to the current GDAL constraints

Reply:
Perfect, just wanted to confirm. We must then check every time, however, if we can release the constrain when newer gdal/geokit versions come out.